app/views/pages/incompany.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1>InCompany</h1>
-
+<%= render 'shared/topbanner' %>
 <%= render 'shared/getready_incompany' %>
 <%= render 'shared/calculate_costs' %>
-<%= render 'shared/incompany_topbanner' %>
\ No newline at end of file
+<%= render 'shared/incompany_topbanner' %>
```

Comment:
If the file names are going to have incompany referenced in them, then I think we should be consistent about whether it is prepended or appended.